README.Rmd

```diff
@@ -1,183 +0,0 @@
----
-output: github_document
----
-<!-- README.md is generated from README.Rmd. Please edit that file, and remember to `knit` when finished -->
```

Comment:
Did you intend to remove README? Or is this an artifact of #127?

Reply:
that is an artifact of 127 https://github.com/2DegreesInvesting/PACTA_analysis/pull/127/files